.idea/dictionaries/Caleb.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="Caleb">
```

Comment:
Not sure what kind of dictionary this is but we should change it from "Caleb" to something more relevant or delete this file all together if this is dummy code.